### PR TITLE
Apache Pulsar 2.7.2 release notes

### PR DESCRIPTION
**Pull request by eolivelli**
This PR supersedes #10477 as GitHub messed up with the diff

### CODE REVIEW

**Comment by eolivelli** [comment]
@Anonymitaet can you please merge this page ?

As I created a new PR your previous Approval is to be confirmed here, and we have the rule that there is at least one approval on the PR

File: site2/website/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,93 @@
...
+- Upgrade Jetty libraries to 9.4.39.v20210325 [#10177](https://github.com/apache/pulsar/pull/10177)
+
+#### Proxy
+- Issue 10221: Fix authorization error while using proxy and `Prefix` subscription authentication mode [#10226](https://github.com/apache/pulsar/pull/10226)
+
+#### Pulsar Admin
+- Add get version command for pulsar rest api, pulsar-admin, pulsar-client [#9975](https://github.com/apache/pulsar/pull/9975)
+
+#### Pulsar SQL
+- Using pulsar SQL query messages will appear `NoSuchLedger… [#9910](https://github.com/apache/pulsar/pull/9910)
```
Comment (zymap) [inline site2/website/release-notes.md:51]:
```suggestion
- Using pulsar SQL query messages will appear NoSuchLedger… [#9910](https://github.com/apache/pulsar/pull/9910)
```

Comment (eolivelli) [reply site2/website/release-notes.md]: thanks @zymap I have applied your suggestion

**Comment by eolivelli** [comment]
@zymap can you please help me merging this patch ?

**Comment by eolivelli** [comment]
@Anonymitaet can you please review again ?

**Review by codelipenghui — APPROVED** [review]

**Review by zymap — APPROVED** [review]

**Pull request merged**